plugins/dell/README.md

```diff
@@ -16,6 +16,23 @@ These devices uses custom GUIDs for Dell-specific hardware.
 
 In both cases the `system-id` is derived from the SMBIOS Product SKU property.
 
+TPM GUIDs are also built using the TSS properties
+`TPM2_PT_FAMILY_INDICATOR`, `TPM2_PT_MANUFACTURER`, and `TPM2_PT_VENDOR_STRING_*`
+These are built hierarchically with more parts for each GUID:
+ * `DELL-TPM-$FAMILY-$MANUFACTURER-$VENDOR_STRING_1`
+ * `DELL-TPM-$FAMILY-$MANUFACTURER-$VENDOR_STRING_1$VENDOR_STRING_2`
+ * `DELL-TPM-$FAMILY-$MANUFACTURER-$VENDOR_STRING_1$VENDOR_STRING_2$VENDOR_STRING_3`
+ * `DELL-TPM-$FAMILY-$MANUFACTURER-$VENDOR_STRING_1$VENDOR_STRING_2$VENDOR_STRING_3$VENDOR_STRING_4`
+
+If there are non-ASCII values in any vendor string or any vendor is missing that octet will be skipped.
+
+Example resultant GUIDs from a real system containing a TPM from Nuvoton:
+```
+  Guid:                 7d65b10b-bb24-552d-ade5-590b3b278188 <- DELL-TPM-2.0-NTC-NPCT
```

Comment:
I'm really wondering if we want to drop the `DELL-` prefix.

Reply:
I considered this too with the thought that there might be a generic flashing mechanism in the future but the binaries expected by Dell plugin are very different than that of what would go into a generic plugin.

So a transition later on would mean spinning them differently.